packages/storybook/.storybook/preview.js

```diff
@@ -1,5 +1,7 @@
-import { configureActions } from '@storybook/addon-actions';
-import { SyntaxHighlighter } from '@storybook/components';
+// eslint-disable-next-line import/no-unresolved
+import { configureActions } from 'storybook/actions';
+// eslint-disable-next-line import/no-unresolved
+import { SyntaxHighlighter } from 'storybook/internal/components';
```

Comment:
we should understand / doublecheck what is going on here. Do we need to file issues? Why are these disables needed?

Reply:
Nice find!